bin/fin

```diff
@@ -4711,66 +4719,75 @@ _exec ()
 	fi
 
 	# Allow overriding container, where to run
+	local container_name
 	if [[ "$1" =~ "--in" ]]; then
-		CONTAINER_NAME=$(echo "$1" | cut -d "=" -f 2)
+		container_name=$(echo "$1" | cut -d "=" -f 2)
 		shift
+	else
+		container_name="cli"
 	fi
 
-	CONTAINER_NAME=${CONTAINER_NAME:-cli}
-	container_id=$(get_container_id "$CONTAINER_NAME")
+	container_id=$(get_container_id "$container_name")
+
+	# Use sh as default shell
+	local SHELL_INTERACTIVE="sh -ilc"
```

Comment:
Use lowercase variable names for local variables, to keep things consistent

Reply:
It is a constant. Hence uppercase